src/Standards/Squiz/Sniffs/WhiteSpace/SuperfluousWhitespaceSniff.php

```diff
@@ -192,6 +192,7 @@ public function process(File $phpcsFile, $stackPtr)
 
             // Ignore blank lines if required.
             if ($this->ignoreBlankLines === true
+                && trim($tokens[$stackPtr]['content']) === ''
```

Comment:
Might be easier to just use `&& $tokens[$stackPtr]['code'] === T_WHITESPACE`

Reply:
Done. (Commit amended)